src/actions/pause/commands.js

```diff
@@ -164,7 +164,8 @@ export function astCommand(stepType: CommandType) {
     }
 
     if (stepType == "stepOver") {
-      const frame = getTopFrame(getState());
+      // This type definition is ambiguous:
+      const frame: any = getTopFrame(getState());
```

Comment:
I coerced this to `any`, as the types coming in were really ambiguous. I didn't think this patch should be the one to sort those types out.